packages/fxa-content-server/app/scripts/lib/router.js

```diff
@@ -302,9 +302,15 @@ Router = Router.extend({
     'report_signin(/)': createViewHandler(ReportSignInView),
 
     'reset_password(/)': function () {
+      const service =
+        Url.searchParam('sync', this.window.location.search) === 'sync'
```

Comment:
I think we may want to pass all the params like `context` as well right or maybe this alone is fine? If this is all we need I think you can just rip this off the relier instead, like `this.relier.get('service')`.

I actually could use this in what I'm working on, so I'll circle back in the morning so I can maybe rebase here after merge.